crates/syntax/src/ast/make.rs

```diff
@@ -158,6 +160,52 @@ fn ty_from_text(text: &str) -> ast::Type {
     ast_from_text(&format!("type _T = {text};"))
 }
 
+/** Related goto [link](https://doc.rust-lang.org/reference/items/type-aliases.html)
+    Type Alias syntax is
+
+    ```
+    TypeAlias :
+       type IDENTIFIER GenericParams? ( : TypeParamBounds )? WhereClause? ( = Type WhereClause?)? ;
+    ```
+
+    FIXME : ident should be of type ast::Ident
+*/
+pub fn ty_alias(
+    ident: String,
```

Comment:
```suggestion
    ident: &str,
```
since ownership of the string isnt required for the function, its generally more ergonomic for callers if this function takes a `&str`, as `&String` coerces to that as well (+ it saves on required cloning quite often)